xm-commons-flow/src/main/java/com/icthh/xm/commons/flow/service/CodeSnippetService.java

```diff
@@ -0,0 +1,66 @@
+package com.icthh.xm.commons.flow.service;
+
+import com.icthh.xm.commons.config.domain.Configuration;
+import com.icthh.xm.commons.exceptions.BusinessException;
+import com.icthh.xm.commons.flow.domain.flow.Flow;
+import com.icthh.xm.commons.flow.domain.flow.Step;
+import com.icthh.xm.commons.flow.domain.flow.Step.Snippet;
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import com.icthh.xm.commons.tenant.TenantContextHolder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static com.icthh.xm.commons.config.client.utils.Utils.nullSafeList;
+import static com.icthh.xm.commons.config.client.utils.Utils.nullSafeMap;
+import static com.icthh.xm.commons.flow.service.CodeSnippetExecutor.SNIPPET;
+import static java.util.stream.Collectors.toList;
+
+@Component
+@LepService(group = "flow")
+public class CodeSnippetService {
+
+    public static final String $$ = "$$";
```

Comment:
propose to use alfanumeric name for constant

Reply:
1 - what name do you suggest?
2 - line 58 will be so long

Reply:
1 - SEGMENT_SEP